### ISSUE TITLE
Link between devportal and connection

### ISSUE THREAD

**Issue by ujovlado** [keboola/developer-portal-ui#357]
There's a link in app detail in design prepared by Jack:

![screenshot_2017-11-02_09-47-12](https://user-images.githubusercontent.com/419849/32317056-d51e5d6c-bfb2-11e7-866c-107ed972202f.png)

I have an idea (https://github.com/keboola/developer-portal/issues/159#issuecomment-341353616) how to solve it.

1. click add
2. select region/connection instance (list provided by devportal ui)
3. (auto) redirect to selected instance (e.g. `https://connection.keboola.com/admin/select-project-for-app?appId=keboola.ex-gmail`)
4. select project (similar list as `https://connection.keboola.com/admin/projects`)
5. (auto) redirect to app detail

**Comment by pivnicek** [comment]
That should work.  It would be cool to have a link to the dev-portal app detail page on the component detail page in connection too (https://connection.keboola.com/admin/projects/99999/extractors/whatever-extractor).

For 5, do you mean the app detail in connection or in dev-portal?

**Comment by ujovlado** [comment]
@pivnicek 5, I meant connection, it's about devportal->connection ;)

**Comment by odinuv** [comment]
We should make this part of vendor setting - when creating a new vendor, we give them a testing project anyway, so we only need to store projectId, connection url in vendor and then show link to "Create configuration in testing project"

**Comment by odinuv** [comment]
related to https://github.com/keboola/developer-portal-ui/issues/353

**Comment by ujovlado** [comment]
> We should make this part of vendor setting - when creating a new vendor, we give them a testing project anyway, so we only need to store projectId, connection url in vendor and then show link to "Create configuration in testing project"

Yes, but as a standard Devportal user (even if not logged in), I should be able to add any public app from Devportal to KBC. That is how I understand it from design which Jack prepared.


Example:
While I'm on this screen https://apps.keboola.com/apps/keboola.ex-google-analytics-v4 I can click to button "Add to Keboola Connection" and it should navigate me to project in KBC where I can create new configuration.

Anyway, we can support both (linking for everyone + linking to test project for developers).
